File: Frontend/public/javascripts/passenger.js
app.controller("PassengersController", function ($scope, $http) {
    $scope.passengers.onContinueClick = function () {
        var bookingCode = localStorage.getItem("bookingCode");
        var currentDate = new Date();
        var birthday = new Date($('.tb-input.datetimepickerWidget.date-of-birth').val());
        var age = currentDate.getYear() - birthday.getYear();
        var ageCode;
        if (age < 2)
            ageCode = 2;
        else if (age <= 12)
            ageCode = 1;
        else ageCode = 0;
        $http.post("http://airlinereservation.somee.com/api/v1/Passengers?bookingcode=" + bookingCode +
            "&epithets=" + $scope.passengers.genderType + "&firstname=" + $scope.passengers.firstname +
            "&lastname=" + $scope.passengers.lastname + "&agecode=" + ageCode)
            .success(function (data) {
                if (data == true) {
                    alert("Đặt vé thành công!!!");
                }
                else alert("Đặt vé thất bại!!!")
            }).error(function (error) {
            alert("Có lỗi!!!");
            return null;
        });
    }
});

File: Backend/AirplaneReservationWebService/AirplaneReservationWebService/DTO/Passenger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AirplaneReservationWebService.DTO
{
    public class Passenger
    {
        private string _bookingCode;

        public string BookingCode
        {
            get { return _bookingCode; }
            set { _bookingCode = value; }
        }
        private string _epithets;

        public string Epithets
        {
            get { return _epithets; }
            set { _epithets = value; }
        }
        private string _firstname;

        public string Firstname
        {
            get { return _firstname; }
            set { _firstname = value; }
        }
        private string _lastname;

        public string Lastname
        {
            get { return _lastname; }
            set { _lastname = value; }
        }
        private int _ageCode;

        public int AgeCode
        {
            get { return _ageCode; }
            set { _ageCode = value; }
        }
        
        public Passenger(string bookingcode, string epithets, string firstname, string lastname, int agecode)
        {
            _bookingCode = bookingcode;
            _epithets = epithets;
            _firstname = firstname;
            _lastname = lastname;
            _ageCode = agecode;
        }
        public Passenger()
        {
            
        }
    }
}